nixos/tests/wallabag.nix

```diff
@@ -0,0 +1,71 @@
+import ./make-test.nix ({ pkgs, ...}: {
+  name = "wallabag";
+  meta = with pkgs.stdenv.lib.maintainers; {
+    maintainers = [ nadrieril ];
+  };
+
+  nodes = {
+    # The only thing the client needs to do is curl
+    client = { ... }: { };
+
+    wallabag = { config, pkgs, ... }: {
+      networking.firewall.allowedTCPPorts = [ 80 ];
+
+      services.wallabag.enable = true;
+      services.wallabag.hostName = "wallabag";
+      services.wallabag.conf = ''
+        parameters:
+          domain_name: http://wallabag
+          database_driver: pdo_pgsql
+          database_driver_class: ~
+          database_host: localhost
+          database_port: null
+          database_name: wallabag
+          database_user: wallabag
+          database_password: password
```

Comment:
Is there a way to not put the database password in the store? I'm thinking it might make sense to just pass the path to the wallabag config file instead of passing the whole file here, in a manner similar to `networking.supplicant.*.configFile.path`, for security reasons.